tests/graph_components/validators/test_config_validator.py

```diff
@@ -0,0 +1,805 @@
+from typing import Dict, List, Optional, Set, Text, Any, Tuple, Type
+import re
+
+import pytest
+from _pytest.monkeypatch import MonkeyPatch
+from unittest.mock import Mock
+
+from rasa.engine.graph import GraphComponent, GraphSchema, SchemaNode
+from rasa.graph_components.validators.config_validator import (
+    POLICY_CLASSSES,
+    DefaultV1ConfigValidator,
+    TRAINABLE_EXTRACTORS,
+)
+from rasa.nlu.constants import FEATURIZER_CLASS_ALIAS
+from rasa.nlu.classifiers.diet_classifier import DIETClassifierGraphComponent
+from rasa.nlu.extractors.regex_entity_extractor import (
+    RegexEntityExtractorGraphComponent,
+)
+from rasa.nlu.extractors.crf_entity_extractor import CRFEntityExtractorGraphComponent
+from rasa.nlu.featurizers.sparse_featurizer.lexical_syntactic_featurizer import (
+    LexicalSyntacticFeaturizerGraphComponent,
+)
+from rasa.nlu.featurizers.sparse_featurizer.regex_featurizer import (
+    RegexFeaturizerGraphComponent,
+)
+from rasa.nlu.selectors.response_selector import ResponseSelectorGraphComponent
+from rasa.nlu.tokenizers.whitespace_tokenizer import WhitespaceTokenizerGraphComponent
+from rasa.core.policies.memoization import MemoizationPolicyGraphComponent
+from rasa.core.policies.rule_policy import RulePolicyGraphComponent
+from rasa.core.policies.ted_policy import TEDPolicyGraphComponent
+from rasa.core.policies.policy import PolicyGraphComponent
+from rasa.shared.core.training_data.structures import StoryGraph
+from rasa.shared.core.domain import KEY_FORMS, Domain, InvalidDomain
+from rasa.shared.exceptions import InvalidConfigException
+from rasa.shared.nlu.constants import (
+    ENTITIES,
+    ENTITY_ATTRIBUTE_GROUP,
+    ENTITY_ATTRIBUTE_ROLE,
+    ENTITY_ATTRIBUTE_TYPE,
+    INTENT_RESPONSE_KEY,
+)
+from rasa.shared.nlu.training_data.training_data import TrainingData
+from rasa.shared.nlu.training_data.message import Message
+from rasa.shared.importers.importer import TrainingDataImporter
+
+
+class DummyImporter(TrainingDataImporter):
+    def __init__(
+        self,
+        training_data: Optional[TrainingData] = None,
+        config: Optional[Dict[Text, Any]] = None,
+        domain: Optional[Domain] = None,
+    ):
```

Comment:
```suggestion
    ) -> None:
```

Reply:
... for tests?

Reply:
oh, didn't see that this wasn't a test in the preview, my bad